sources/documents/RTB/Verbali/Esterni/VerbaleEsterno_2024-04-09/capitolo-riunione.tex

```diff
@@ -0,0 +1,137 @@
+
+
+\subsubsection{Pre-prompt}
+
+\textbf{Domanda:} Quindi ci si aspetta un pre-prompt?
+
+\textbf{Risposta:} Quando il dizionario dati è di dimensioni ridotte, come tre tabelle, può essere contenuto interamente all’interno del prompt. Però, se il dizionario è di maggiore dimensione, come mille tabelle, si presentano dei problemi tra cui il costo elevato dell’elaborazione di un prompt con numerosi token. Teoricamente è possibile avere prompt grandi ma è meglio mantenerli piccoli, capendo quali siano le sezioni pertinenti alla richiesta dell’utente o meno. Diventa così necessario capire il metodo con cui minimizzare il dizionario dati durante la creazione del prompt.
+[la proponente esegue una dimostrazione, facendo elaborare ad un LLM un prompt e aggiungendo progressivamente dettagli superflui, è evidente che il modello, all’aggiunta di dati superflui inizi ad allucinare]
+
+\subsubsection{Funzionalità di debug}
+
+\textbf{Domanda:} Il debug è uno stato intermedio rispetto a quanto visto?
+
+\textbf{Risposta:} Sì, il debug è utile a capire come è stato generato il prompt e perché sono presenti determinate porzioni del dizionario dati.
+
+
+\subsubsection{Stima correttezza prompt}
+
+\textbf{Domanda:} Come si può stimare la correttezza del prompt?
+
+\textbf{Risposta:} Si parte dalla richiesta generica dell'utente nella parte finale. Quello che ci interessa è la parte che gli si attacca prima, ovvero la descrizione del dizionario dati attinente alla richiesta. È necessario essere in grado di capire come si è arrivati alla formulazione di quella parte che viene prefissata alla richiesta.
+
+\subsubsection{Visualizzazione debug}
+
+\textbf{Domanda:} Per la visualizzazione del debug, si può sovrascrivere la funzionalità di visualizzazione del prompt ottenuto quando l’attore che lo richiede è un Tecnico. È l’approccio adatto?
+
+\textbf{Risposta:} Un’opzione sarebbe fare la extend del caso normale, che avviene quando il Tecnico è interessato a vedere queste informazioni aggiuntive.
+
+
+\subsubsection{Modalità di visualizzazione per utente tecnico}
+
+\textbf{Domanda:} Il tecnico possiede solo la visualizzazione di debugging o anche quella normale?
+
+
+\textbf{Risposta:} Entrambe, perché sono due cose diverse. Se il dizionario è molto piccolo, la situazione è semplice. Quando invece un dizionario è grande, il prompt può diventare fuorviante se vengono introdotte tabelle che non servono o non ci sono tabelle che servono. Vediamo una controprova.
+[La Proponente esegue delle richieste in cui i dati forniti sono più o meno degli stretti necessari]
+Si osserva che se i dati sono in eccesso, il modello si confonde, quando invece mancano, se li inventa. Il problema è trovare i pezzi minimi per comporre il dizionario dati.
```

Comment:
```suggestion
Si osserva che se i dati sono in eccesso, il modello si confonde, quando invece mancano, genera dati casuali. Il problema è trovare i pezzi minimi per comporre il dizionario dati.
```
Modificherei il termine per restare un pelo più tecnici

Reply:
Modificheresti "se li inventa" in "genera dati casuali" o c'è un termine da modificare ulteriormente?